tokens/usecase/tokens_usecase.go

```diff
@@ -32,24 +32,74 @@ type tokensUseCase struct {
 
 // Struct to represent the JSON structure
 type AssetList struct {
-	ChainName string `json:"chain_name"`
+	ChainName string `json:"chainName"`
 	Assets    []struct {
-		Description string `json:"description"`
-		DenomUnits  []struct {
-			Denom    string `json:"denom"`
-			Exponent int    `json:"exponent"`
-		} `json:"denom_units"`
-		Base     string        `json:"base"`
-		Name     string        `json:"name"`
-		Display  string        `json:"display"`
-		Symbol   string        `json:"symbol"`
-		Traces   []interface{} `json:"traces"`
-		LogoURIs struct {
+		ChainName        string `json:"chainName"`
+		SourceDenom      string `json:"sourceDenom"`
+		CoinMinimalDenom string `json:"coinMinimalDenom"`
+		Symbol           string `json:"symbol"`
+		Decimals         int    `json:"decimals"`
+		LogoURIs         struct {
 			PNG string `json:"png"`
 			SVG string `json:"svg"`
-		} `json:"logo_URIs"`
-		CoingeckoID string   `json:"coingecko_id"`
-		Keywords    []string `json:"keywords"`
+		} `json:"logoURIs"`
+		CoingeckoID string `json:"coingeckoId"`
+		Price       struct {
+			PoolID string `json:"poolId"`
+			Denom  string `json:"denom"`
+		} `json:"price"`
+		Categories   []string `json:"categories"`
+		PegMechanism string   `json:"pegMechanism"`
+		// Found inconsistency in the assest list file
+		// TransferMethods/CounterParty sometimes is an array and sometimes is an object
+		// So commenting out the fields for now
```

Comment:
I'm supportive of removing all fields that we don't need and skip parsing them. Instead, leave the comment that this is what we are trying to do

Reply:
Removed unused asset list definitions